CONTRIBUTORS

```diff
@@ -196,9 +207,12 @@ Yassine TIJANI <ytijani@vmware.com>
 yiyingy <yiyingy@vmware.com>
 ykakarap <yuva2811@gmail.com>
 Yogesh Sobale <6104071+ysobale@users.noreply.github.com>
+Yue Yin <yueyin@yuyin-a01.vmware.com>
 Yun Zhou <yunz@vmware.com>
 Yuya Kusakabe <yuya.kusakabe@gmail.com>
+Zach G <zguan@vmware.com>
 Zach Tucker <ztucker@vmware.com>
+zach96guan <zach96guan@users.noreply.github.com>
```

Comment:
Note that we have https://github.com/vmware/govmomi/blob/master/.mailmap to avoid dups like this, but it's kindof a pain. ok to leave this as-is for now, we can fix it up later.

Reply:
Thanks for pointing this out! Yeah, this looks painful. I've edited the mail map and fixed this.